docs/book/tutorials/kubernetes-on-vsphere-with-kubeadm.md

```diff
@@ -156,262 +208,994 @@ bootstrapTokens:
          - authentication
 nodeRegistration:
   kubeletExtraArgs:
-    cloud-provider: "vsphere"
-    cloud-config: "/etc/kubernetes/vsphere.conf"
+    cloud-provider: external
 ---
 apiVersion: kubeadm.k8s.io/v1beta1
 kind: ClusterConfiguration
+useHyperKubeImage: false
 kubernetesVersion: v1.14.2
-apiServer:
-  extraArgs:
-    cloud-provider: "vsphere"
-    cloud-config: "/etc/kubernetes/vsphere.conf"
-  extraVolumes:
-  - name: cloud
-    hostPath: "/etc/kubernetes/vsphere.conf"
-    mountPath: "/etc/kubernetes/vsphere.conf"
-controllerManager:
-  extraArgs:
-    cloud-provider: "vsphere"
-    cloud-config: "/etc/kubernetes/vsphere.conf"
-  extraVolumes:
-  - name: cloud
-    hostPath: "/etc/kubernetes/vsphere.conf"
-    mountPath: "/etc/kubernetes/vsphere.conf"
 networking:
+  serviceSubnet: "10.96.0.0/12"
   podSubnet: "10.244.0.0/16"
+etcd:
+  local:
+    imageRepository: "k8s.gcr.io"
```

Comment:
Don't think etcd and dns sections are required - remove.

Reply:
Can we just make sure before I do so? These are the steps I followed, and they worked.